sdk-tests/src/test/java/io/dapr/it/actors/ActorTracingIT.java

```diff
@@ -0,0 +1,124 @@
+package io.dapr.it.actors;
+
+import com.jayway.jsonpath.DocumentContext;
+import com.jayway.jsonpath.JsonPath;
+import io.dapr.actors.ActorId;
+import io.dapr.actors.client.ActorProxyBuilder;
+import io.dapr.client.DaprClient;
+import io.dapr.it.BaseIT;
+import io.dapr.it.actors.app.MyActor;
+import io.dapr.it.actors.app.MyActorService;
+import io.dapr.it.tracing.http.OpenTelemetryConfig;
+import io.opentelemetry.api.OpenTelemetry;
+import io.opentelemetry.api.trace.Span;
+import io.opentelemetry.api.trace.SpanKind;
+import io.opentelemetry.api.trace.Tracer;
+import io.opentelemetry.context.Scope;
+import net.minidev.json.JSONArray;
+import okhttp3.HttpUrl;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+import org.junit.jupiter.api.Test;
+
+import java.util.UUID;
+
+import static io.dapr.it.tracing.OpenTelemetry.createOpenTelemetry;
+import static io.dapr.it.tracing.OpenTelemetry.getReactorContext;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class ActorTracingIT extends BaseIT {
+
+    @Test
+    public void testInvoke() throws Exception {
+        var run = startDaprApp(
+            ActorTracingIT.class.getSimpleName()+"Server",
+            MyActorService.SUCCESS_MESSAGE,
+            MyActorService.class,
+            true,
+            60000);
+        var clientRun = startDaprApp(
+            ActorTracingIT.class.getSimpleName()+"Client",
+            60000);
+
+        OpenTelemetry openTelemetry = createOpenTelemetry();
+        Tracer tracer = openTelemetry.getTracer(OpenTelemetryConfig.TRACER_NAME);
+        String spanName = UUID.randomUUID().toString();
+        Span span = tracer.spanBuilder(spanName).setSpanKind(SpanKind.CLIENT).startSpan();
+
+        try (DaprClient client = run.newDaprClientBuilder().build()) {
+            MyActor myActor =
+                new ActorProxyBuilder<>(
+                    "MyActorTest",
+                    MyActor.class,
+                    clientRun.newActorClient()).build(new ActorId("123456"));
+            try (Scope scope = span.makeCurrent()) {
+                client.waitForSidecar(10000).block();
+                myActor.say("hello world")
+                    .contextWrite(getReactorContext(openTelemetry))
+                    .block();
+            }
+        } finally {
+            span.end();
+        }
```

Comment:
This is an example showing how the client app should invoke actors and tracing headers to work.